template/src/{{packagename}}/__version__.py.jinja

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.0"
```

Comment:
It was attempted to get the version from `importlib.metadata.version` in `__init__.py.jinja`.

But let's stick to what is suggested here unless you would like to play around and get the other way working.